src/app/context-menu-builder.ts

```diff
@@ -433,6 +435,21 @@ export class ContextMenuBuilder {
     return menu;
   }
 
+  /**
+   * Adds the Paste menu item.
+   */
+  public addPasteAsPlainText(menu, menuInfo) {
+    menu.append(
+      new MenuItem({
+        label: this.stringTable.pasteAsPlainText(),
+        accelerator: 'CommandOrControl+Shift+V',
+        enabled: menuInfo.editFlags.canPaste,
+        role: 'pasteAndMatchStyle',
```

Comment:
is this suppose to be pasteAsPlainText?

Reply:
yes, which correspond to role "pasteAndMatchStyle" on chromium